src/database/FirehoseConnector.js

```diff
@@ -147,9 +147,15 @@ class FirehoseConnector {
                     receiverTracks,
                     senderTracks
                 },
-                transportAggregates: { meanRtt }
+                transportAggregates: { meanRtt },
+                inboundVideoExperience: {
+                    principalVideoAggregates,
+                    secondaryVideoAggregates
+                }
```

Comment:
The "principal video" keeps track of the frame rate/resolution of the dominant inbound video element (the one with the best resolution/frame rate) and it aims to represent the upper bound of the video experience of the user at any given point in time. The "secondary video" is for the lower bound experience. Does it make sense to you guys? Also, I am not very happy with the names I have chosen for these two variables, so I am open to suggestions @andrei-gavrilescu @nils-ohlmeier :-) Maybe instead of the prefix "principal" we should use "high" or "upperBound"?

Reply:
Just from the names I assumed that "principal" refers to on stage big resolution. And "secondary" refers to the grid view resolution. So I guess that suggests that we should rename it :-)

So yeah I think "upperBound" and "lowerBound" make more sense to me.